packages/paste-website/src/pages/patterns/navigation/index.mdx

```diff
@@ -0,0 +1,276 @@
+- **Navigating within a product feature** should be handled at the page-level, with components like [In-Page Navigation](/components/in-page-navigation) and [Breadcrumb](/components/breadcrumb).
+
+<ResponsiveImage src={NavigationWireframe} aria-hidden="true" alt="" />
+
+### Accessibility
+
+The Sidebar and Topbar navigation components have the following considerations:
+
+- Ensure the [Skip links](https://www.w3.org/TR/UNDERSTANDING-WCAG20/navigation-mechanisms-skip.html) are [correctly implemented](/components/sidebar#skip-links). This ensures users can easily and quickly access the main content of a web page by bypassing repeated blocks of content such as navigation links.
+- Ensure the `SidebarNavigation` component is labelled with an accessible name.
+
+### Topbar ingredients
+
+<ResponsiveImage src={NavigationTopbar} aria-hidden="true" alt="" />
+
+- [Topbar](/components/topbar)
+- [Account Switcher](/components/account-switcher)
+- [Anchor](/components/anchor)
+- [Badge](/components/badge)
+- [Combobox](/components/combobox)
+- [Icons](/components/icons), particularly `ChevronDownIcon`, `SearchIcon`, `SupportIcon`, `UserIcon`
+- [Menu](/components/menu)
+- [Product Switcher](/components/product-switcher)
+- [Status Menu](/components/status-menu)
+- [User Dialog](/components/user-dialog)
+
+### Sidebar ingredients
+
+<ResponsiveImage src={NavigationSidebar} aria-hidden="true" alt="" />
+
+- [Sidebar container](/components/sidebar)
+- [Sidebar Navigation](/components/sidebar-navigation)
+- [Icons](/components/icons), mainly product and feature logo icons
+
+## Examples
+
+### Product-agnostic
+
+<StoryPreview
+  title="Generic docs full composition"
+  storyID="components-sidebar-fullcompositions--docs"
+  code={`
+<Sidebar
+  sidebarNavigationSkipLinkID={sidebarNavigationSkipLinkID}
+  topbarSkipLinkID={topbarSkipLinkID}
+  mainContentSkipLinkID={mainContentSkipLinkID}
+  collapsed={true}
+  variant="compact"
+>
+  <SidebarHeader>
+    <SidebarHeaderIconButton as="a" href="#">
+      <LogoTwilioIcon size="sizeIcon20" decorative={false} title="Go to Console homepage" />
+    </SidebarHeaderIconButton>
+    <SidebarHeaderLabel>Twilio Console</SidebarHeaderLabel>
+  </SidebarHeader>
+  <SidebarBody>
+  </SidebarBody>
+  <SidebarFooter>
+    <SidebarCollapseButton
+      i18nCollapseLabel="Close sidebar"
+      i18nExpandLabel="Open sidebar"
+    />
+  </SidebarFooter>
+</Sidebar>
+<SidebarPushContentWrapper collapsed={true} variant="compact">
+  <Button variant="primary">
+    Toggle Push Sidebar
+  </Button>
+</SidebarPushContentWrapper>`}
+/>
+
+<Box marginBottom="space70">
+  <Callout variant="neutral">
+    <CalloutHeading>Do you work on a Twilio product platform team?</CalloutHeading>
+    <CalloutText>
+      Navigation and information architecture are ever-evolving. Collaborate with us on defining more specific
+      guidelines for your product by [starting a GitHub Discussion](https://github.com/twilio-labs/paste/discussions).
```

Comment:
todo: can't use markdown in jsx 

<img width="709" alt="image" src="https://github.com/twilio-labs/paste/assets/1592327/e31b26c1-6ba1-483d-a5d3-5e5d01c59703">